specifyweb/context/app_resource.py

```diff
@@ -22,13 +22,31 @@
 DIR_LEVELS = ['Personal', 'UserType', 'Collection', 'Discipline', 'Common', 'Backstop']
 
 # At the discipline level of the hierarchy, filesystem resources are found in
-# the directories defined in "disciplines.xml".
-disc_file = os.path.join(settings.SPECIFY_CONFIG_DIR, "disciplines.xml")
-
-disciplines = ElementTree.parse(disc_file)
-
-discipline_dirs = dict( (disc.attrib['name'], disc.attrib.get('folder', disc.attrib['name']))
-    for disc in disciplines.findall('discipline') )
+# the directories were defined in "disciplines.xml" but are now defined here.
+DISCIPLINE_DIRS = {
+    "fish": "fish",
+    "herpetology": "herpetology",
+    "paleobotany": "invertpaleo",
+    "invertpaleo": "invertpaleo",
+    "vertpaleo": "vertpaleo",
+    "bird": "bird",
+    "mammal": "mammal",
+    "insect": "insect",
+    "botany": "botany",
+    "invertebrate": "invertebrate",
+    "minerals": "minerals",
+    "anthropology": "anthropology",
+    # "vascplant": "vascplant",
+    # "fungi": "fungi",
+}
+
+FORM_RESOURCE_EXCLUDED_LST = [
+    "fish/fishbase.views.xml",
+    "accessions/accessions.views.xml",
+    "backstop/system.views.xml",
+    "backstop/editorpanel.views.xml",
+    "backstop/gbif.views.xml",
+]
```

Comment:
I presume this is what is excluding those five form definitions from the list of choices
Effectively for an admin user, this filters out the five form definitions:

**Now hidden:**
- Fish > Fishbase
- Accessions
- Backstop > Editorpanel
- Backstop > Gbif
- Backstop > System

Reply:
I suggest creating a new app resource for each form definition and ensuring that they contain forms that apply to Specify 7.

After reviewing some of the forms this morning, I found one that is not hidden but should be (under the name 'Backstop > Preferences' in the app).
We need to add least add this:

```py
    "backstop/preferences.views.xml",
```

Reply:
I'll add 'backstop/preferences' to the exclude list, let me know if there are anymore to add